AppSyncRealTimeClientTests/Connection/AppSyncSubscriptionConnectionErrorHandlerTests.swift

```diff
@@ -67,7 +68,7 @@ class AppSyncSubscriptionConnectionErrorHandlerTests: XCTestCase {
             case .failed(let error):
                 guard let connection = error as? ConnectionProviderError,
                       case .subscription(let id, _) = connection,
-                      id != nil else {
+                      !id.isEmpty else {
```

Comment:
Did the `id` associated value type for `.subscription` change? I don't understand how `id != nil` and `!id.isEmpty` could both compile with the same type.

Reply:
TIL that's a warning not an error.